File: examples/modulith-clinical-tool-jpa/src/main/java/io/zenwave360/example/clinicaltool/modules/clinical/core/domain/events/DoctorCreated.java
package io.zenwave360.example.clinicaltool.modules.clinical.core.domain.events;

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*;
import jakarta.validation.constraints.*;
import java.io.Serializable;
import java.math.*;
import java.time.*;
import java.util.*;

/**
 *
 */
@lombok.Getter
@lombok.Setter
@org.springframework.modulith.NamedInterface("events.DoctorCreated")
public class DoctorCreated implements Serializable {

    @java.io.Serial
    private static final long serialVersionUID = 1L;

    private Doctor doctor;
}

File: examples/modulith-klinical-tool-jpa/src/main/kotlin/io/zenwave360/example/clinicaltool/modules/clinical/core/domain/events/DoctorCreated.kt
package io.zenwave360.example.clinicaltool.modules.clinical.core.domain.events

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*
import jakarta.validation.constraints.*
import java.io.Serializable
import java.math.*
import java.time.*
import java.util.*

/**  */
@org.springframework.modulith.NamedInterface("events.DoctorCreated")
data class DoctorCreated(var doctor: Doctor? = null) : Serializable {

    companion object {
        private const val serialVersionUID: Long = 1L
    }
}
